src/components/SearchBox/SearchBox.test.tsx

```diff
@@ -5,18 +5,23 @@ import SearchBox from "./SearchBox";
 
 describe("SearchBox ", () => {
   it("renders", () => {
-    const wrapper = shallow(<SearchBox />);
+    const wrapper = shallow(<SearchBox onSearch={jest.fn()} />);
     expect(wrapper).toMatchSnapshot();
   });
 
   it("shows the clear button when there is a value", () => {
-    const wrapper = shallow(<SearchBox value="admin" />);
+    const wrapper = shallow(<SearchBox onSearch={jest.fn()} value="admin" />);
     expect(wrapper.find(".p-search-box__reset").exists()).toBe(true);
   });
 
   it("can externally control the value", () => {
     const wrapper = shallow(
-      <SearchBox externallyControlled onChange={jest.fn()} value="admin" />
+      <SearchBox
+        externallyControlled
+        onSearch={jest.fn()}
+        onChange={jest.fn()}
```

Comment:
There doesn't appear to be a test that checks that this method is called when we expect it to.